contracts/Tokens/VTokens/VToken.sol

```diff
@@ -511,6 +533,55 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return _setInterestRateModelFresh(newInterestRateModel);
     }
 
+    /**
+     * @notice Accrues interest and updates the stable interest rate model using _setStableInterestRateModelFresh
+     * @dev Admin function to accrue interest and update the stable interest rate model
+     * @param newStableInterestRateModel The new interest rate model to use
+     * @return uint 0=success, otherwise a failure (see ErrorReporter.sol for details)
+     * custom:events Emits NewMarketInterestRateModel event; may emit AccrueInterest
+     * custom:events Emits NewMarketStableInterestRateModel, after setting the new stable rate model
+     */
+    function setStableInterestRateModel(StableRateModel newStableInterestRateModel) public returns (uint256) {
+        // Check caller is admin
+        if (msg.sender != admin) {
+            return fail(Error.UNAUTHORIZED, FailureInfo.SET_STABLE_INTEREST_RATE_MODEL_OWNER_CHECK);
+        }
+
+        uint error = accrueInterest();
+        if (error != uint(Error.NO_ERROR)) {
+            // accrueInterest emits logs on errors, but on top of that we want to log the fact that an attempted change of interest rate model failed
+            return fail(Error(error), FailureInfo.SET_INTEREST_RATE_MODEL_ACCRUE_INTEREST_FAILED);
+        }
+
+        // We fail gracefully unless market's block number equals current block number
+        if (accrualBlockNumber != getBlockNumber()) {
+            return fail(Error.MARKET_NOT_FRESH, FailureInfo.SET_STABLE_INTEREST_RATE_MODEL_FRESH_CHECK);
+        }
+
+        // _setInterestRateModelFresh emits interest-rate-model-update-specific logs on errors, so we don't need to.
+        return _setStableInterestRateModelFresh(newStableInterestRateModel);
+    }
+
+    /**
+     * @notice Returns the current per-block borrow interest rate for this vToken
+     * @return rate The borrow interest rate per block, scaled by 1e18
```

Comment:
@notice and @return is same as `borrowRatePerBlock`. Here we should specify "borrow stable interest rate" and for `borrowRatePerBlock` we should specify "variable".

Reply:
Fixed in [commit](https://github.com/VenusProtocol/venus-protocol/pull/210/commits/482b814841c2e68904e01250a89f48d5c745e83a)